src/System/Relay/isRequestCacheable.ts

```diff
@@ -15,3 +17,10 @@ export const hasNoCacheParamPresent = url => {
 
   return false
 }
+
+// Important - Add any new personalized argument checks to this list. That way, logged-in queries
```

Comment:
This was just my initial stab - we have this one case we want to cover - when this argument is present and truth-y.

I figure we can refine this lever if/when we have more.